docs/deployment/deploy_shared_data.md

```diff
@@ -2,7 +2,7 @@
 
 This topic describes how to deploy and use a shared-data StarRocks cluster.
 
-The shared-data StarRocks cluster is specifically engineered for the cloud on the premise of separation of storage and compute. It allows data to be stored in object storage that is compatible with the S3 protocol (for example, AWS S3 and MinIO). You can achieve not only cheaper storage and better resource isolation, but elastic scalability for your cluster. The query performance of the shared-data StarRocks cluster aligns with that of a classic cluster in case of local cache hits.
+The shared-data StarRocks cluster is specifically engineered for the cloud on the premise of separation of storage and compute. It allows data to be stored in object storage that is compatible with the S3 protocol (for example, AWS S3 and MinIO). You can achieve not only cheaper storage and better resource isolation, but elastic scalability for your cluster. The query performance of the shared-data StarRocks cluster aligns with that of a classic cluster in case of local disk cache hits.
```

Comment:
The query performance of the shared-data StarRocks cluster aligns with that of a classic cluster (shared-nothing) when the query hits the local disk cache.